docs/doc_v1.0/Build_documentation.md

```diff
@@ -381,97 +465,103 @@ Figure 11. Infection map example
 
 ### Run Contact Data
 
-Running the propagation of infection is the heart of the calculation.
-Figure 12 shows the flow of data.
+The runContactData method within Outbreak.java is where the main calculations in the code are triggered. 
+
+## Event Runner
+
+The events within the model are handled by a group of lists that contain specific events that can occur in the model. These are:
+
+1.	Alert Event
+2.	Contact Event
+3.	Infection Event
+4.	Virus Event
```

Comment:
Since this is a numbered list, perhaps the order should reflect the order they are called in the code.

```suggestion
1.	Virus Event
2.	Infection Event
3.	Contact Event
4.	Alert Event
```